hw/ip/flash_ctrl/dv/env/flash_ctrl_env.core

```diff
@@ -1,48 +1,49 @@
-CAPI=2:
-# Copyright lowRISC contributors.
-# Licensed under the Apache License, Version 2.0, see LICENSE for details.
-# SPDX-License-Identifier: Apache-2.0
-name: "lowrisc:dv:flash_ctrl_env:0.1"
-description: "FLASH_CTRL DV UVM environment"
-filesets:
-  files_dv:
-    depend:
-      - lowrisc:dv:ralgen
-      - lowrisc:dv:dv_base_reg
-      - lowrisc:dv:dv_lib
-      - lowrisc:dv:cip_lib
-      - lowrisc:dv:mem_bkdr_util
-      - lowrisc:ip:flash_ctrl_pkg
-      - lowrisc:constants:top_pkg
-    files:
-      - flash_ctrl_env_pkg.sv
-      - flash_ctrl_if.sv
-      - flash_mem_addr_attrs.sv: {is_include_file: true}
-      - flash_ctrl_seq_cfg.sv: {is_include_file: true}
-      - flash_ctrl_env_cfg.sv: {is_include_file: true}
-      - flash_ctrl_env_cov.sv: {is_include_file: true}
-      - flash_ctrl_virtual_sequencer.sv: {is_include_file: true}
-      - flash_ctrl_scoreboard.sv: {is_include_file: true}
-      - flash_ctrl_env.sv: {is_include_file: true}
-      - seq_lib/flash_ctrl_vseq_list.sv: {is_include_file: true}
-      - seq_lib/flash_ctrl_callback_vseq.sv: {is_include_file: true}
-      - seq_lib/flash_ctrl_base_vseq.sv: {is_include_file: true}
-      - seq_lib/flash_ctrl_common_vseq.sv: {is_include_file: true}
-      - seq_lib/flash_ctrl_rand_ops_base_vseq.sv: {is_include_file: true}
-      - seq_lib/flash_ctrl_smoke_vseq.sv: {is_include_file: true}
-      - seq_lib/flash_ctrl_rand_ops_vseq.sv: {is_include_file: true}
-    file_type: systemVerilogSource
-
-generate:
-  ral:
-    generator: ralgen
-    parameters:
-      name: flash_ctrl
-      ip_hjson: ../../data/flash_ctrl.hjson
-
-targets:
-  default:
-    filesets:
-      - files_dv
-    generate:
-      - ral
+CAPI=2:

```

Comment:
Oops! it seems you are using windows, and have lines ending with the classical extra \r\n. Please use Unix line conventions.

See https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md#posix-file-endings.